docs/whirlpool/docs/05-Legacy SDK/01-Developer Portal.md

```diff
@@ -0,0 +1 @@
+https://orca-so.gitbook.io/orca-developer-portal
```

Comment:
Instead of a link can we add the actual page (so that we can decommission the old developer portal)?

Reply:
This is meant as a placeholder for the docs of the Legacy SDK. I will remove it from the sidebar for now.